index.html

```diff
@@ -100,48 +100,48 @@ <h3 class="article__title">How can I help?</h3>
                     Learn more
                   </a>
                 </p>
-              </div>
-            </div>
+              </article>
+            </section>
           </div>
-        </div>
+        </main>
 
-        <div class="sidebar">
+        <aside class="sidebar">
           <h2 class="heading-underline">Upcoming events</h2>
 
-          <div class="article">
+          <article class="article">
             <img
               class="article__thumbnail"
-              src="spring-fundraisers_thumbnail.jpg"
+              src="images/spring-fundraisers_thumbnail.jpg"
               alt=""
             />
             <div class="article__content">
               <h3 class="article__title">
-                <a class="article__title-link" href="#">Join us for our Spring fundraisers</a>
+                <a class="article__title-link" href="#">Join us for our Spring <br/>fundraisers</a>
               </h3>
               <p class="article__summary">
                 Riders needed now to help raise awareness and funds for BfR and
                 Médecins Sans Frontières.
               </p>
             </div>
-          </div>
+          </article>
 
-          <div class="article">
+          <article class="article">
             <img
               class="article__thumbnail"
-              src="bikes-for-refugees_logo.jpg"
+              src="images/bikes-for-refugees_logo.jpg"
               alt=""
             />
             <div class="article__content">
               <h3 class="article__title">
-                <a class="article__title-link" href="#">Bike dropoff event in Edinburgh on May 11</a>
+                <a class="article__title-link" href="#">Bike dropoff event in <br/>Edinburgh on May 11</a>
```

Comment:
Adding hard-coded break points is not usually encouraged. Would you be able to solve the issue with e.g. setting `max-width`?